### PR TITLE
Arbab/edxapp main rewrite

### PR DESCRIPTION
**Pull request by arbabnazar**
@edx/devops kindly review. Thanks

Jenkins build to verify this branch (7508)

### CODE REVIEW

**Comment by arbabnazar** [comment]
@smarnach please also review this one as you are reviewing the other part of this change. Thanks

File: playbooks/roles/edxapp/tasks/main.yml [hunk 1]
```diff
@@ -3,72 +3,71 @@
...
+    - { path: "{{ edxapp_app_dir }}/.ssh" }
+    - { path: "{{ edxapp_venvs_dir }}" }
+    - { path: "{{ edxapp_theme_dir }}" }
+    - { path: "{{ edxapp_staticfile_dir }}" }
+    - { path: "{{ edxapp_course_static_dir }}" }
+    - { path: "{{ edxapp_course_data_dir }}" }
+    # var should have more permissive permissions than the rest
+    - { path: "{{ edxapp_data_dir }}", mode: "0775" }
+    # directory to import the courses from github
+    - { path: "{{ EDXAPP_GIT_REPO_DIR }}", mode: "0775" }
```
Comment (smarnach) [inline playbooks/roles/edxapp/tasks/main.yml:34]: The task to create the last directory doesn't have any tags in the original version (which is probably a mistake in the old version).

**Comment by smarnach** [comment]
@arbabnazar This looks all good to me.  I've started deploying a sandbox using a merge of #3075, #3076 and #3079, and I will see how it goes.  I'll let you know once I know more.

**Comment by smarnach** [comment]
@arbabnazar The sandbox deployment using the merged branch of all three PRs worked fine, and I can't detect any problems with it, so all good from my side.

**Comment by e0d** [comment]
:+1:

**Pull request merged**